lib/lexisnexis/request.rb

```diff
@@ -78,7 +78,17 @@ def url_request_path
     end
 
     def uuid
-      attributes.fetch(:uuid, SecureRandom.uuid)
+      uuid = attributes.fetch(:uuid, SecureRandom.uuid)
+
+      if uuid_prefix
+        "#{uuid_prefix}:#{uuid}"
+      else
+        uuid
+      end
```

Comment:
Two ideas:

1. should we do a presence/emptiness check? So that passing `""` doesn't mean we get a leading colon?
2. Maybe one of these to reduce branching?

```
[uuid_prefix, uuid].select(&:present?).join(':')
```